src/components/DisplayChapterInfo.tsx

```diff
@@ -60,56 +61,66 @@ const DisplayChapterInfo = ({
       </div>
       {userRequests && (
         <div className="mb-12">
-          <h1 className="mb-6 text-xl">{`Pending (${userRequests.length})`}</h1>
-          {userRequests.length > 0 ? (
-            <CardGrid
-              column_count={2}
-              tiles={userRequests.map((user, index) => {
-                return (
-                  <PendingCard
-                    key={index}
-                    name={fullName(user.user)}
-                    uid={user.id}
-                  />
-                );
-              })}
-            />
-          ) : (
-            <h1 className="font-light">
-              {"This chapter has no pending members."}
-            </h1>
-          )}
+          <DropDownContainer
+            title={
+              <div className="mb-6 text-xl">{`Pending (${userRequests.length})`}</div>
+            }
+          >
+            {userRequests.length > 0 ? (
+              <CardGrid
+                column_count={2}
+                tiles={userRequests.map((user, index) => {
+                  return (
+                    <PendingCard
+                      key={index}
```

Comment:
Yep, I agree. In general, it's safer to not use index, as items in a list can be shifted / re-ordered. I think in this case, it's fine, because `userRequests` are passed from the server, and we refresh the page every time. You can find more details [here](https://stackoverflow.com/a/59518082)